src/main/java/types/data/Event.java

```diff
@@ -0,0 +1,40 @@
+package types.data;
+
+import utilities.DateTimeParser;
+
+import java.time.LocalDateTime;
+
+import static types.data.Deadline.DATE_TIME_FORMATTER;
+
+/**
+ * Event data type.
+ */
+public class Event extends Task {
+    private final LocalDateTime start;
+    private final LocalDateTime end;
+
+    private Event(String n, String d1, String d2) {
+        super(n, "E");
+        start = DateTimeParser.parse(d1);
+        end = DateTimeParser.parse(d2);
+    }
+
+    /**
+     * Factory method to get a new Event object.
+     * @param n Description.
+     * @param d1 Starting date or time.
+     * @param d2 Ending date or time.
+     * @return Newly created Event object.
+     */
+    public static Event create(String n, String d1, String d2) {
+        return new Event(n, d1, d2);
+    }
```

Comment:
Might want to consider better naming for parameters so as to follow the coding standard?